oauth2_provider/models.py

```diff
@@ -6,7 +6,7 @@
 from django.conf import settings
 from django.core.urlresolvers import reverse
 from django.db import models, transaction
-from django.utils import timezone
+from django.utils import timezone, six, six
```

Comment:
Considering DOT already depends on `six` being available, it would make the most sense to just `import six` instead.